lib/src/test/java/io/ably/lib/test/rest/RestAuthTest.java

```diff
@@ -673,8 +673,8 @@ public Object getTokenRequest(TokenParams params) throws AblyException {
 			/* Publish a message */
 			Message messagePublishee = new Message(
 					"wildcard",	// name
-					"brian that is called brian",	// clientId
-					String.valueOf(System.currentTimeMillis())	// data
+					String.valueOf(System.currentTimeMillis()), "brian that is called brian"    // clientId
+					// data
 			);
```

Comment:
This looks wrong, given the signature is `name, data, clientId`, then your comments of `// name, //clientId` is wrong?  How did this pass if so or have I misread this?